app/assets/stylesheets/application/components/check-box.scss

```diff
@@ -0,0 +1,103 @@
+.check-box > .info {
+  @include display(flex);
+
+  > .left {
+    @include flex-basis(40px);
+    @include flex-shrink(0); // 横幅を40pxに固定する
+
+    width: 40px;
+  }
+
+  > .right {
+    margin-left: 10px;
+    width: 100%;
+  }
+
+  > .right > .work {
```

Comment:
Merge rule `> .right > .work` with rule on line 11<br>Selector should have depth of applicability no greater than 3, but was 4